src/v2/view-builder/views/email/OTPEmailView.js

```diff
@@ -48,21 +48,30 @@ const Body = BaseForm.extend(Object.assign(
   {
     save: loc('mfa.challenge.verify', 'login'),
 
-    subtitle: 'To finish signing in, click the link in your email or enter code below.',
+    subtitle:
+    'An email has been sent to you. Please click the link in your email or enter the code from that email below.',
 
     saveForm () {
       BaseForm.prototype.saveForm.apply(this, arguments);
       this.stopPolling();
       // TODO: abort ongoing request. (https://oktainc.atlassian.net/browse/OKTA-244134)
     },
 
+    sendEmailLink () {
+      _.delay(_.bind(function () {
```

Comment:
add comment why we do this and how you choose `300ms` (guessing randomly?)

Reply:
Yeah, no particular reason. Just waiting for the form to finishing rendering and start polling.